mapper/javabean/src/main/java/org/hibernate/search/mapper/javabean/session/SearchSessionBuilder.java

```diff
@@ -17,6 +18,12 @@
 	 */
 	SearchSessionBuilder tenantId(String tenantId);
 
+	/**
+	 * @param commitStrategy The commit strategy for indexing works added to the {@link SearchSession#getMainWorkPlan() main work plan}.
+	 * @return {@code this} for method chaining.
+	 */
+	SearchSessionBuilder commitStrategy(DocumentCommitStrategy commitStrategy);
```

Comment:
Not really. I really need to turn this into a real module that can be used in the real world, so that we can add tests that make sense. There's a ticket somewhere... In the meantime I just added this option for consistency.